ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -572,4 +572,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = 97C146E61CF9000F007C117D /* Project object */;
-}
\ No newline at end of file
+}
```

Comment:
I see you built the app for iOS, did you have to change/configure something or did it Just Work™?

Reply:
I built it on my mac for iPhone 13 emulator with no problems and with no changes :) I also built it on my physical android phone.